SIG.md

```diff
@@ -0,0 +1,108 @@
+# Cilium SIGS
+
+The Cilium project has Special Interest Groups, or SIGs, with a common purpose of advancing the project with respect to a specific topic, such as network policy or documentation. Our goal is to enable a distributed decision structure and code ownership, as well as providing focused forums for getting work done, making decisions, and onboarding new contributors. 
+
+Areas covered by SIGs may be vertically focused on particular components or functions, cross-cutting/horizontal, spanning many/all functional areas of the project, or in support of the project itself. Examples:
+- Policy, Scaling, Tetragon, Hubble, Service Mesh
+
+
+Each SIG must have a charter that specifies its scope (topics, subsystems, code repos, and/or directories), responsibilities, areas of authority, how members and roles of authority/leadership are selected/granted, how decisions are made, and how conflicts are resolved. See the SIG charter process for details on how charters are managed. SIGs are relatively free to customize or change how they operate, within some broad guidelines and constraints imposed by cross-project processes (e.g., the release process) and assets (e.g., the cilium repo).
+
+A primary reason that SIGs exist is as forums for collaboration. Much work in a SIG should stay local within that SIG. However, SIGs must communicate in the open, ensure other SIGs and community members can find notes of meetings, discussions, designs, and decisions, and periodically communicate a high-level summary of the SIG's work to the community.
+
+# SIG Roles
+## SIG Lead
+
+- Membership tracked in sigs.yaml
+- Must be an odd number of SIG Leads in a group
+- MUST be a review of one or more areas of the project, and should be a subject matter expert in their SIG area
+- SHOULD define how priorities and commitments are managed. MAY delegate to other leads as needed.
+- SHOULD drive charter changes (including creation) to get community buy-in but MAY delegate content creation to SIG contributors.
+- MUST identify, track, and maintain the metadata of the SIGs enhancements for the current release and serve as point of contact for the release team, but MAY delegate to other contributors to fulfill these responsibilities.
+- MUST organize a main group meeting and make sure sigs.yaml is up to date
+- SHOULD facilitate meetings but MAY delegate to other members or future chairs/chairs in training.
+- MUST ensure meetings are recorded and made at least notes made available.
+- MUST coordinate communication and be a connector with other community groups like SIGs and the committers but MAY delegate the actual communication and creation of content to other contributors where appropriate.
+
+### Requirements
+Chairs MUST be at least a "member" on our contributor ladder to be eligible to hold a leadership role within a SIG.
+SIGs MAY prefer various levels of domain knowledge depending on the role. This should be documented.
+
+
+### On-boarding and Off-boarding Leads
```

Comment:
```suggestion
### On-boarding and Off-boarding Chairs
```

Reply:
i thought we were sticking with the term "Leads"?

Reply:
Yes, I thought we had decided on Lead. Either way we should get consistent in this doc

Reply:
You're right, but everything below is about chairs so I got confused. Can you make it consistent?